scripts/configure.py

```diff
@@ -114,6 +114,32 @@
       'sudo rm /var/log/user*     && echo "removed user logs"   || echo ok',
     ]
   },
+  'support_tunnel': {
+    'apt': [
+      'libsystemd-dev'
+    ],
+    'copy' : [{
+       'from': 'config/support_tunnel_crontab',
+       'to': '/etc/cron.d/support_tunnel'
+    }],
+    'script': [
+      'sudo mkdir -p /var/lib/support_tunnel',
+      'sudo chmod 0777 /var/lib/support_tunnel', # TODO: lock this down
+      'if [ ! -e /opt/support_tunnel ] ; then'
+      '  pushd $(mktemp --directory)',
+      '  git clone https://github.com/micro-nova/support_tunnel.git',
+      '  sudo mv support_tunnel /opt',
+      '  popd',
+      'fi',
+      'pushd /opt/support_tunnel',
+      'git pull',
```

Comment:
What happens when we rewrite history? Maybe this is a little safer? `git fetch && git reset --hard main` I don't know of a better way to get latest safely.